app/attributes/attributecontroller.cpp

```diff
@@ -904,6 +920,64 @@ void AttributeController::recalculateDerivedItems( bool isFormValueChange, bool
     }
   }
 
+  // Evaluate form items editability
```

Comment:
Isn't it more like 
```suggestion
  // Evaluate if form items are editable
```
? :)